### PR TITLE
Fix inclusive language issues

### PR DESCRIPTION
**Pull request by apinnick**
* Replace blacklist and whitelist according to inclusive language guidelines.
* Only generic text is changed.
* Command blocks are unchanged.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_configuring-ipxe-to-reduce-provisioning-times.adoc
Comment (ekohl) [inline guides/common/modules/proc_configuring-ipxe-to-reduce-provisioning-times.adoc:7]: I dislike the allowlist/denylist terminology. Can't we be more generic here and simply say that is somehow unreachable? Because that's the core of the issue.
```suggestion
* An unreachable PXE service, for example by a firewall restriction.
```

File: guides/common/modules/ref_skipping-steps-when-performing-backups.adoc
Comment (ekohl) [inline guides/common/modules/ref_skipping-steps-when-performing-backups.adoc:1]: Related context: https://github.com/theforeman/foreman_maintain/pull/723 is open to change the name. Not because of "inclusive" issues but because it's not really what's going on. Normally a whitelist means "only apply this and nothing else" but here it really is implemented as "skip these steps". Essentially it's a blacklist.

File: guides/common/modules/proc_configuring-http-proxy-to-connect-to-cdn.adoc
Comment (apinnick) [reply guides/common/modules/proc_configuring-http-proxy-to-connect-to-cdn.adoc:14]: Will add to Contributing.

Re: backticks. I might do that if the lead-in sentence said, "To run the `subscription-manager` command with the HTTP proxy..." However, general usage makes more sense in this context.

File: guides/common/modules/proc_configuring-ipxe-to-reduce-provisioning-times.adoc
Comment (apinnick) [reply guides/common/modules/proc_configuring-ipxe-to-reduce-provisioning-times.adoc:7]: Works for me.

File: guides/common/modules/ref_skipping-steps-when-performing-backups.adoc
Comment (apinnick) [reply guides/common/modules/ref_skipping-steps-when-performing-backups.adoc:1]: Agree. Perhaps best to hold off on merging this PR until your PR is merged.

File: guides/common/modules/ref_skipping-steps-when-performing-backups.adoc
Comment (maximiliankolb) [reply guides/common/modules/ref_skipping-steps-when-performing-backups.adoc:5]: still open.

File: guides/common/modules/proc_configuring-http-proxy-to-connect-to-cdn.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-http-proxy-to-connect-to-cdn.adoc:10]: also still open.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
two threads remain open; rest LGTM

**Comment by apinnick** [comment]
Looks like I forgot to push the commit. Done

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by Lennonka** [comment]
Cherry-picked:
- 4f45441c5d..0642a5c357  3.9 -> 3.9

**Pull request merged**
